backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,100 @@
+create table project
+(
+    project_id  bigserial     not null primary key,
+    name        varchar(256)  not null unique,
+    description varchar(1000) null
+);
+
+create table test
+(
+    test_id bigserial    not null primary key,
+    name    varchar(256) not null unique
+);
+
+create table subtest
+(
+    subtest_id      bigserial      not null primary key,
+    test_id         bigint         not null,
+    name            varchar(256)   not null unique,
+    -- Method
+    meth            varchar(10000) null,
+    confluence_link varchar(1000)  null,
+    constraint belongs_to_test
+        foreign key (test_id) references test (test_id)
+            on update cascade
+            on delete cascade
+);
+
+-- Corresponds to an EXCEL file
+create table experiment
+(
+    experiment_id bigserial                not null primary key,
+    project_id    bigint                   not null,
+    subtest_id    bigint                   not null,
+    -- This is input from the user
+    name          varchar(256)             null,
+    meta          varchar(1000)            null,
+    -- This is fallback name to show if name is not specified
+    raw_file_name varchar(256)             not null,
+    upload_date   timestamp with time zone not null,
+
+    constraint belongs_to_project
+        foreign key (project_id) references project (project_id)
+            on update cascade
+            on delete cascade,
+
+    constraint has_subtest
+        foreign key (subtest_id) references subtest (subtest_id)
+            on update cascade
+            -- Prevent removing subtest if there is an experiment having such subtest
+            on delete no action
+);
+
+create table measurement
+(
+    measurement_id bigserial        not null primary key,
+    experiment_id  bigint           not null,
+    -- Either compound id or name
+    compound_id    varchar(256)     not null,
+    concentration  double precision not null,
+    signal         double precision not null,
+
+    constraint belongs_to_experiment
+        foreign key (experiment_id) references experiment (experiment_id)
+            on delete cascade
+            on update cascade
+);
+
+-- We can remove this table if we don't need anything apart from IC50
```

Comment:
After we got some new data about the current scope of development and decided to focus on experimental data storage rather than analysis stuff, I removed most of analysis generalization from my document (product specification aka discovery report). I think we can really assume IC50, apparently we won't need this generalization any soon.

Reply:
As we figured out on the last meeting, there might be different kinds of IC50 as well. So we agreed on having this table